exo-go/features/run/run.feature

```diff
@@ -0,0 +1,66 @@
+Feature: running Exosphere applications
+
+  As an Exosphere developer
+  I want to have an easy way to run an application on my developer machine
+  So that I can test my app locally.
+
+  Rules:
+  - run "exo run" in the directory of your application to run it
+  - this command boots up all the services of the application
+
+
+  Scenario: booting a functioning Exosphere application
+    Given a running "simple" application
+    Then my machine is running the services:
+      | NAME |
+      | web  |
+
+
+  Scenario: booting a complex Exosphere application
+    Given a running "running" application
+    Then my machine is running ExoCom
+    And my machine is running the services:
+      | NAME  |
+      | web   |
+      | users |
+    # And ExoCom uses this routing:
```

Comment:
> YAML is just a superset of JSON, so to avoid such crazy YAML constructs, simply use JSON syntax on this line.

I got rid of this test in #333 but to address this comment, we might need to change how `exo-setup` (or `exo-run` soon) renders `SERVICE_ROUTES`.
@hugobho am I right?